src/main/groovy/com/mini/asaas/utils/Validators.groovy

```diff
@@ -0,0 +1,72 @@
+package com.mini.asaas
+
+class Validators {
+
+    static boolean validateByType(String cpfCnpj) {
```

Comment:
se for mudar o nome da classe para `CpfCnpjUtils` só validade aqui fica claro, se não precisa especificar na assinatura do método o que ele valida